webapp/src/main/webapp/css/news.less

```diff
@@ -337,6 +337,7 @@
       }
 
     .summary {
```

Comment:
I think that it would be better to put that on newsDetails-description instead in order to consider title, summary and body